finder-app/Makefile

```diff
@@ -1,28 +1,19 @@
+CC = aarch64-none-linux-gnu-gcc  # Use ARM64 cross-compiler
```

Comment:
@Sriramz2002 this change will mean you can no longer compile for the host, would be better to continue to use CROSS_COMPILE here.